rules/klayout/lvs/gf180mcu.lvs

```diff
@@ -255,13 +255,15 @@ if $schematic
   schematic($schematic, reader)
   logger.info("Netlist file: #{$schematic}")
 else
-  begin  
-    schematic("#{source.cell_name}.cdl", reader)
-    logger.info("Netlist file: #{source.cell_name}.cdl")
-  rescue 
-    puts "No schematic loaded , please add your netlist file"
-    logger.error("No schematic loaded , please add your netlist file")
-  end 
+  exts = ["ngspice", "spice", "cdl"]
+  candidates = exts.map{|ext| "#{source.cell_name}.#{ext}"}  
```

Comment:
@proppy Could you please remove 'ngspice' as we can't use the netlist used for simulation for LVS?

Reply:
@proppy ?